tests/behat/features/insert-a-link.feature

```diff
@@ -33,7 +33,7 @@ So that I can link to a external website or a page on my site
     And I wait for 1 second
     And I select "youranchor" from "Form_EditorToolbarLinkForm_AnchorSelector"
     And I press the "Insert link" button
-    Then the "Content" HTML field should contain "<a href="[sitetree_link,id=3]#youranchor">awesome</a>"
+    Then the "Content" HTML field should contain /<a href="\[sitetree_link,id=[0-9]+\]#youranchor">awesome</a>/
```

Comment:
You know, I'm wondering whether we could split this up so it doesn't look at IDs at all. With a regex check like this we aren't concerned about "3" any more anyway... How about this:

```behat
Then the "Content" HTML field should contain "<a href="[file_link,id=""
And the "Content" HTML field should contain "#youranchor"
And the "Content" HTML field should contain "awesome"
```

This way we wouldn't need the framework regex change at all